libs/sdk-backend-tiger/src/convertors/fromBackend/dateFormatting/dateValueParser.ts

```diff
@@ -10,9 +10,9 @@ const granularityParseValueTransformations: {
     [granularity in DateAttributeGranularity]?: ValueTransform;
 } = {
     "GDC.time.day_in_week": (value) => {
-        // server returns 0 = Sunday, 6 = Saturday
-        // date-fns expects 1 = Monday, 7 = Sunday
-        return value === "0" ? "7" : value;
+        // server returns 00 = Sunday, 06 = Saturday
+        // date-fns expects 1 = Sunday, 7 = Saturday
+        return `${parseInt(value) + 1}`;
     },
 };
 
```

Comment:
Maybe some link to https://date-fns.org/docs/format or https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table would be useful here (so it's clear, which standard it uses)